### DISCUSSION TITLE
Allow simpler modification of weapon damage values

### DISCUSSION THREAD

**Discussion by NotAFile** [piqueserver/piqueserver#636] [Feature Requests]
This issue was recently raised in OS: yvt/openspades#558

I disagree with us tweaking the values ourselves as this is a very political decision. I do however see the value of being able to change this more easily than is currently possible.

IMO the ideal solution is for use to have the damage settings in a seperate file and activate the "piqueserver recommended" settings files that nerf the SMG by default but still have a .75 settings file available. This would be a config option, so you can select the damage file you want.

**Comment by kmsiapps** [comment]
How about this script? It's from Korean forum, and I can ask the author for permission if we need this.
[weaponprofile.zip](https://github.com/piqueserver/piqueserver/files/1442931/weaponprofile.zip)

**Comment by NotAFile** [comment]
well, it's nice for sure. But I thought the idea was to make it easier internally? Currently you need to patch constants.WEAPONS
